android/app/build.gradle

```diff
@@ -196,6 +196,7 @@ android {
 }
 
 dependencies {
+    implementation project(':@react-native-community_cameraroll')
```

Comment:
Is this really needed? We use RN 0.60+ and it automatically linking libraries.

Reply:
@djorkaeffalexandre Sorry, my bad.